artifacthub/library/general/poddisruptionbudget/1.0.0/template.yaml

```diff
@@ -36,7 +36,10 @@ spec:
         violation[{"msg": msg}] {
           obj := input.review.object
           pdb := data.inventory.namespace[obj.metadata.namespace]["policy/v1"].PodDisruptionBudget[_]
-          obj.spec.selector.matchLabels == pdb.spec.selector.matchLabels
+
+          matchLabels := { [label, value] | some label; value := pdb.spec.selector.matchLabels[label] }
```

Comment:
@nilekhc should artifacthub 1.0.0 artifacts be untouched? these changes should only go to the 1.0.1 folder right?

Reply:
The entire `/artifact` directory should be untouched. We should update the template only inside `/src` with the correct version.  After that run `make generate-artifacthub-artifacts` and it will auto-update `/artifact` directory.

Reply:
It looks like the `generate-artifacthub-artifacts` must have been run before updating the template version. This should go in 1.0.1

Reply:
Woops, forgot to correct that.